config.py

```diff
@@ -20,9 +20,12 @@
 NER_FACILITY_MODEL_FT = '/app/ner_models/ner_facility_ft/ner_model_facility/'
 RUN_NER_FACILITIES_AFTER_EXTRACTION = False
 
+NLP_MODEL = 'en_core_web_sm'
```

Comment:
I did not compare the different models because `en_core_web_sm` is the only one out of the three that doesn't try to kill the celery workers, and shows high-performance measures.